ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs

```diff
@@ -145,8 +144,11 @@ nodeToNodeCodecCBORTerm _version
 
 data ConnectionMode = UnidirectionalMode | DuplexMode
 
+
 -- | Check whether a version enabling diffusion pipelining has been
 -- negotiated.
+--
+-- TODO: this ought to be defined in `ouroboros-consensus` or `ouroboros-consensus-node`
```

Comment:
This comment says `ouroboros-consensus-node`, but the packaged is named `ouroboros-node`